ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/StandaloneExecutor.java

```diff
@@ -114,37 +135,126 @@ private void showWelcomeMessage() {
     writer.flush();
   }
 
+
+  private void initFunctionMap() {
+    functionMap.put(SetProperty.class, (Pair<String, Statement> statementPair) -> {
+      final SetProperty setProperty = (SetProperty) statementPair.getRight();
+      configProperties.put(setProperty.getPropertyName(), setProperty.getPropertyValue());
+      return null;
+    });
+
+    functionMap.put(UnsetProperty.class, (Pair<String, Statement> statementPair) -> {
+      final UnsetProperty unsetProperty = (UnsetProperty) statementPair.getRight();
+      configProperties.remove(unsetProperty.getPropertyName());
+      return null;
+    });
+
+    functionMap.put(CreateStream.class, (Pair<String, Statement> statementPair) -> {
+      ksqlEngine.buildMultipleQueries(statementPair.getLeft(), ksqlConfig, configProperties);
+      return null;
+    });
+
+    functionMap.put(CreateTable.class, (Pair<String, Statement> statementPair) -> {
+      ksqlEngine.buildMultipleQueries(statementPair.getLeft(), ksqlConfig, configProperties);
+      return null;
+    });
+
+    functionMap.put(CreateStreamAsSelect.class, (Pair<String, Statement> statementPair) -> {
+      handlePersistentQuery(statementPair.getRight(), statementPair.getLeft(), configProperties);
+      return null;
+    });
+
+    functionMap.put(CreateTableAsSelect.class, (Pair<String, Statement> statementPair) -> {
+      handlePersistentQuery(statementPair.getRight(), statementPair.getLeft(), configProperties);
+      return null;
+    });
+
+    functionMap.put(InsertInto.class, (Pair<String, Statement> statementPair) -> {
+      handlePersistentQuery(statementPair.getRight(), statementPair.getLeft(), configProperties);
+      return null;
+    });
+  }
+
+
   private void executeStatements(final String queries) {
-    final List<QueryMetadata> queryMetadataList = ksqlEngine.createQueries(queries, ksqlConfig);
-    for (QueryMetadata queryMetadata : queryMetadataList) {
-      if (queryMetadata instanceof PersistentQueryMetadata) {
-        PersistentQueryMetadata persistentQueryMetadata = (PersistentQueryMetadata) queryMetadata;
-        persistentQueryMetadata.start();
+    final List<Pair<String, Statement>> statementPairs =
+        ksqlEngine.parseStatements(queries, ksqlEngine.getMetaStore().clone());
+    for (final Pair<String, Statement> statementPair: statementPairs) {
+      final String statementString = statementPair.getLeft();
+      final Statement statement = statementPair.getRight();
+      if (functionMap.containsKey(statement.getClass())) {
+        functionMap.get(statement.getClass()).apply(statementPair);
       } else {
         final String message = String.format(
             "Ignoring statements: %s"
-            + "%nOnly CREATE statements can run in standalone mode.",
-            queryMetadata.getStatementString()
+                + "%nOnly DDL (CREATE STREAM/TABLE, DROP STREAM/TABLE, SET, UNSET) "
```

Comment:
Why are we allowing `DROP` statements in non-interactive mode?

Reply:
You may want to first clean the metastore and then run the queries to recreate the streams/tables. Clickstream demo does this.

Reply:
But if you're running headless... isn't the metastore empty to begin with? So ... do you need `DROP`?

Reply:
So you said above we're dropping DROP, right?

Reply:
Yes, `DROP` is not supported. I also added a test for this.